kalamine/layout.py

```diff
@@ -203,6 +204,21 @@ def __init__(self, filepath: Path) -> None:
 
         # keyboard layers: self.layers & self.dead_keys
         rows = GEOMETRY[self.meta["geometry"]].rows
+
+        # Adding Angle Mod support
+        if self.is_angle_mod:
+            not_compatible_with_angle_mod = True
+            for row in rows:
+                if row.keys[0] == "lsgt":
+                    not_compatible_with_angle_mod = False
+                    row.keys[:6] = [row.keys[5]] + row.keys[
+                        :5
+                    ]  # ['ab05', 'lsgt', 'ab01', 'ab02', 'ab03', 'ab04']
```

Comment:
nitpick : please put this comment on its own line above line 214, so it doesn’t force the formatter to split the code onto three lines